lib/stack_master/commands/apply.rb

```diff
@@ -64,7 +64,8 @@ def create_stack
           failed!("Stack creation aborted")
         end
         upload_files
-        cf.create_stack(stack_options.merge(tags: proposed_stack.aws_tags))
+        on_failure = @config.stack_defaults['on_failure'] || 'ROLLBACK'
```

Comment:
To me this would be most useful when creating a fresh stack. I only want to delete the stack if it's the first time I create it because then I can't update it.

So in my use case I see no value, I see the most value in a command line flag. But that's just my use case :)